cmd/main.go

```diff
@@ -56,7 +56,7 @@ func main() {
 	initCmd := initcmd.NewCmdInit(f, ioStreams)
 	updateHelp(names, initCmd)
 	loader := manifestreader.NewManifestLoader(f)
-	invFactory := inventory.ClusterClientFactory{}
+	invFactory := inventory.ClusterClientFactory{StatusPolicy: inventory.StatusPolicyAll}
```

Comment:
do we need this if the default ConfigMap inventory doesn't store the status anyway? That's why i had left it to default to None before.

Reply:
We injected the apply status into the ConfigMap inventory. Updated the InventoryCM, now we don't need to change this to All.